scipy/stats/tests/test_morestats.py

```diff
@@ -619,38 +619,116 @@ def test_1d_input(self):
 
 
 class TestBinomP(object):
+    """Tests for stats.binom_test."""
```

Comment:
Do you think it would be worthwhile to ensure that `binomtest` passes all the tests of `binom_test`?
Since it's a class, you could replace calls to `stats.binom_test` with `self.binomial_test_function` or similar, and make subclasses with `self.binomial_test_function = stats.binom_test` and `self.binomial_test_function = stats.binomtest`.

Reply:
I added this in the recent update.